app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -87,6 +94,23 @@ class MainActivity : AppCompatActivity() {
     fun onSuperBackPressed() {
         super.onBackPressed()
     }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+
+        if (BuildConfig.FLAVOR == getString(R.string.playStore) && requestCode == RC_CREDENTIALS_READ) {
```

Comment:
Define as const in the class itself, not in the resources file.

Reply:
updated